src/User/Access/AbstractPolicy.php

```diff
@@ -58,7 +58,22 @@ public function checkAbility(User $actor, string $ability, $instance)
 
         // If a "total access" method is defined, try that.
         if (method_exists($this, 'can')) {
-            return call_user_func_array([$this, 'can'], [$actor, $ability, $instance]);
+            return $this->sanitizeResult(call_user_func_array([$this, 'can'], [$actor, $ability, $instance]));
         }
     }
+
+    /**
+     * @param mixed $result
+     * @return string|void
+     */
+    public function sanitizeResult($result)
+    {
+        if ($result === true) {
+            $result = $this->allow();
```

Comment:
You could directly `return` on lines 72 and 74.

Not sure if there's any performance improvement to that but the assignment seems unnecessary :sweat_smile: